src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/SignedXml.cs

```diff
@@ -33,7 +33,7 @@ public class SignedXml
         private int[] _refLevelCache = null;
 
         internal XmlResolver _xmlResolver = null;
-        internal XmlElement _context = null;
+        private XmlElement _context = null;
```

Comment:
This change seems to have been mainly aesthetic, and since there're still internal fields (_xmlResolver) this isn't even really getting an argument of fixing an inconsistency.  It does add inconsistency, in that the code is less like desktop now.

So please undo the addition of the Context property.